src/fetch_tse_data.py

```diff
@@ -0,0 +1,254 @@
+    "DESC_SIT_TOT_TURNO",#
+    "NM_EMAIL",#
+]
+
+header_consulta_cand_from2014=[
+    "DATA_GERACAO", 
+    "HORA_GERACAO",
+    "ANO_ELEICAO",
+    "NUM_TURNO",# (*)
+    "DESCRICAO_ELEICAO",# (*)
+    "SIGLA_UF",
+    "SIGLA_UE",# (*)
+    "DESCRICAO_UE",
+    "CODIGO_CARGO",# (*)
+    "DESCRICAO_CARGO",
+    "NOME_CANDIDATO",
+    "SEQUENCIAL_CANDIDATO",# (*)
+    "NUMERO_CANDIDATO",
+    "CPF_CANDIDATO",
+    "NOME_URNA_CANDIDATO",
+    "COD_SITUACAO_CANDIDATURA",
+    "DES_SITUACAO_CANDIDATURA",
+    "NUMERO_PARTIDO",
+    "SIGLA_PARTIDO",
+    "NOME_PARTIDO",
+    "CODIGO_LEGENDA",
+    "SIGLA_LEGENDA",
+    "COMPOSICAO_LEGENDA",
+    "NOME_LEGENDA",
+    "CODIGO_OCUPACAO",
+    "DESCRICAO_OCUPACAO",
+    "DATA_NASCIMENTO",
+    "NUM_TITULO_ELEITORAL_CANDIDATO",
+    "IDADE_DATA_ELEICAO",
+    "CODIGO_SEXO",
+    "DESCRICAO_SEXO",
+    "COD_GRAU_INSTRUCAO",
+    "DESCRICAO_GRAU_INSTRUCAO",
+    "CODIGO_ESTADO_CIVIL",
+    "DESCRICAO_ESTADO_CIVIL",
+    "CODIGO_COR_RACA",
+    "DESCRICAO_COR_RACA",
+    "CODIGO_NACIONALIDADE",
+    "DESCRICAO_NACIONALIDADE",
+    "SIGLA_UF_NASCIMENTO",
+    "CODIGO_MUNICIPIO_NASCIMENTO",
+    "NOME_MUNICIPIO_NASCIMENTO",
+    "DESPESA_MAX_CAMPANHA",
+    "COD_SIT_TOT_TURNO",
+    "DESC_SIT_TOT_TURNO",
+    "NM_EMAIL",
+]
+
+
+sel_columns=[
+# "DATA_GERACAO", 
+# "HORA_GERACAO",
+"ANO_ELEICAO",
+"NUM_TURNO",# (*)
+"DESCRICAO_ELEICAO",# (*)
+"SIGLA_UF",
+#"SIGLA_UE",# (*)
+"DESCRICAO_UE",
+# "CODIGO_CARGO",# (*)
+"DESCRICAO_CARGO",
+"NOME_CANDIDATO",
+"SEQUENCIAL_CANDIDATO",# (*)
+# "NUMERO_CANDIDATO",
+"CPF_CANDIDATO",
+# "NOME_URNA_CANDIDATO",
+# "COD_SITUACAO_CANDIDATURA",
+# "DES_SITUACAO_CANDIDATURA",
+# "NUMERO_PARTIDO",
+# "SIGLA_PARTIDO",
+# "NOME_PARTIDO",
+# "CODIGO_LEGENDA",
+# "SIGLA_LEGENDA",
+# "COMPOSICAO_LEGENDA",
+# "NOME_LEGENDA",
+# "CODIGO_OCUPACAO",
+# "DESCRICAO_OCUPACAO",
+# "DATA_NASCIMENTO",
+"NUM_TITULO_ELEITORAL_CANDIDATO",
+# "IDADE_DATA_ELEICAO",
+# "CODIGO_SEXO",
+# "DESCRICAO_SEXO",
+# "COD_GRAU_INSTRUCAO",
+# "DESCRICAO_GRAU_INSTRUCAO",
+# "CODIGO_ESTADO_CIVIL",
+# "DESCRICAO_ESTADO_CIVIL",
+# "CODIGO_COR_RACA",
+# "DESCRICAO_COR_RACA",
+# "CODIGO_NACIONALIDADE",
+# "DESCRICAO_NACIONALIDADE",
+# "SIGLA_UF_NASCIMENTO",
+# "CODIGO_MUNICIPIO_NASCIMENTO",
+# "NOME_MUNICIPIO_NASCIMENTO",
+# "DESPESA_MAX_CAMPANHA",
+# "COD_SIT_TOT_TURNO",
+"DESC_SIT_TOT_TURNO",
+# "NM_EMAIL",
+]
+
+
+###Concatenate all files in one pandas dataframe
+cand_df=pd.DataFrame()
+for year in year_list:
+    filesname=FILENAME_PREFIX+year+'*.txt'
+    filespath=os.path.join(TEMP_PATH,filesname)
+    files_of_the_year= sorted(glob.glob(filespath))
+    for file_i in files_of_the_year:
+        # the following cases do not take into account next elections. hopefully, TSE will add headers to the files
+        if ('2014' in file_i) or ('2016' in file_i):
+            cand_df_i=(pd.read_csv('./'+file_i,sep=';',header=None,dtype=str,names=header_consulta_cand_from2014,encoding='iso-8859-1'))
```

Comment:
Actually, I used "unicode" in python 2.7. In python 3, the standard str replaced unicode. We can use np.str as long as it deals with unicode characters. Does it?